notes/extended-profile.md

```diff
@@ -0,0 +1,18 @@
+# Loading the full profile
+
+It is possible to have all profile information in a single profile document but a more likely situation is that some information is in the profile document and some is in extended profile documents.  One key reason for this is that the profile document is a public document and therefore resources and prefernces that are private need to be described in private documents.  The public profile document becomes the starting point for a "follow-your-nose" process to look at the private and other extended profile documents. 
```

Comment:
```suggestion
It is possible to have all profile information in a single profile document but a more likely situation is that some information is in the profile document and some are in extended profile documents.  One key reason for this is that the profile document is a public document and therefore resources and preferences that are private need to be described in private documents.  The public profile document becomes the starting point for a "follow-your-nose" process to look at the private and other extended profile documents. 
```

Reply:
I believe that since information is a mass noun (some information is equivalent to some sugar, we can have "less" information not "fewer" informations) that the verb should be singular.  We do not say "some of the sugar are rancid".